index.html

```diff
@@ -305,6 +305,35 @@ <h2 class="section-title mt-0">Stay in the know</h2>
                     </div>
                 </div>
             </section>
+
+            <section class="contributors section">
+                <div class="container">
+                    <div class="text-center">
+                        <h2 class="section-title mt-0">Our Dream Team</h2>
+                        <div class="contributors-wrapper">
+                            <div class="contributors-row">
+                                <div class="contributors-column technical">
+                                    <svg height="48" id="processor" viewBox="0 0 48 48" width="48" xmlns="http://www.w3.org/2000/svg"><defs></defs><rect class="vi-primary" height="24" width="24" x="12" y="12"/><path class="vi-accent" d="M16,16H32V32H16V16ZM8,16h4v3H8V16Zm0,6h4v3H8V22Zm0,6h4v3H8V28ZM36,16h4v3H36V16Zm0,6h4v3H36V22Zm0,6h4v3H36V28ZM32,8v4H29V8h3ZM26,8v4H23V8h3ZM20,8v4H17V8h3ZM31,36v4H28V36h3Zm-6,0v4H22V36h3Zm-6,0v4H16V36h3Z"/></svg>
+                                    <div class="contributor-list">
+                                        <p>Ashley Narcisse</p>
+                                        <p>Takashi Wickes</p>
+                                        <p>Blake Pfaff</p>
```

Comment:
would love for these to be anchor tags to link to the contributor's github! 🎁 (adding this comment to the `contributing.md` file as well)

Reply:
Ashely Nacrisse: https://github.com/darkfadr
Takashi Wickes: https://github.com/takashiw
Blake Pfaff: https://github.com/blakecarson

Reply:
Are they technical or non-technical contributors?

Reply:
We're going to be keeping track on the README the types of contributions people are making according to [`allcontributors`](https://allcontributors.org/docs/en/emoji-key). These are their types of contributions! ✨ 

Ashley: `🚇 Infrastructure`
Takashi: `🎨 Design`
Blake: ` 📖 Documentation`